### ISSUE TITLE
clang-10 warning: definition of implicit copy constructor is deprecated

### ISSUE THREAD

**Issue by k15tfu** [boostorg/phoenix#97]
In boost/phoenix/core/actor.hpp:
```
include/boost/phoenix/core/actor.hpp:148:9: error: definition of implicit copy constructor for 'actor<boost::log::v2s_mt_posix::expressions::aux::unary_function_terminal<boost::log::v2s_mt_posix::expressions::has_attribute<void> > >' is deprecated because it has a user-declared copy assignment operator [-Werror,-Wdeprecated-copy]
        BOOST_PROTO_EXTENDS_ASSIGN()
        ^
```

**Comment by Kojoley** [comment]
This is the Proto issue, it should silence the warning inside the macro.

**Comment by nickwilliams-zaxiom** [comment]
I'm still getting this warning in 1.80.0 (which includes #98), the warning has just moved to the added line:

```
phoenix/core/actor.hpp:228:39: warning: definition of implicit copy constructor for 'actor<boost_MW610beta2::proto::exprns_::basic_expr<boost_MW610beta2::proto::tagns_::tag::terminal, boost_MW610beta2::proto::argsns_::term<boost_MW610beta2::boost_MW610beta2phoenix::detail::rethrow>, 0>>' is deprecated because it has a user-declared copy assignment operator [-Wdeprecated-copy]
        BOOST_DELETED_FUNCTION(actor& operator=(actor const&))
```

Do I need to file a new bug? Or is there already a different bug I didn't find?

**Comment by nickwilliams-zaxiom** [comment]
And right as I commented, I found #114.

**Comment by Kojoley** [comment]
> I'm still getting this warning in 1.80.0 (which includes #98), the warning has just moved to the added line:
> 
> ```
> phoenix/core/actor.hpp:228:39: warning: definition of implicit copy constructor for 'actor<boost::proto::exprns_::basic_expr<boost::proto::tagns_::tag::terminal, boost::proto::argsns_::term<boost::boost_phoenix::detail::rethrow>, 0>>' is deprecated because it has a user-declared copy assignment operator [-Wdeprecated-copy]
>         BOOST_DELETED_FUNCTION(actor& operator=(actor const&))
> ```
> 
> Do I need to file a new bug? Or is there already a different bug I didn't find?

It is fixed in 1.81 by https://github.com/boostorg/phoenix/pull/108
